server/lib/pg_client.js

```diff
@@ -0,0 +1,15 @@
+const pg = require('pg');
+require('env2')('config.env');
+
+const pgClient = (connection) => {
+  const connectionString = process.env.DATABASE_URL || connection;
+  const client = new pg.Client(connectionString);
+  client.connect((err) => {
```

Comment:
I don't think so. Or else the client being returned would be within the client.connect callback.